python/README.md

```diff
@@ -1,107 +0,0 @@
-# Mosaic by Databricks Labs
```

Comment:
@edurdevic why is this file deleted?

Reply:
This was an outdated copy of the main README.